galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/internal/HttpClientImpl.java

```diff
@@ -341,6 +350,35 @@ private void addHeaders(AbstractHttpMessage message, ContentType contentType, Co
 
     }
 
+    public void getFile(String destination, String path) {
```

Comment:
Destination should be a java.nio.file.Path,  you are assuming the destination must be a local filesystem.   It could be used for the RAS.